.spelling

```diff
@@ -129,6 +133,7 @@ TAAR
 TBD
 TCP
 timestamp
+Timestamp
```

Comment:
A lowercase word in the `.spelling` file is case-insensitive, so you shouldn't have to add `Timestamp` if `timestamp` is already there.